src/Bicep.Core/Emit/PlaceholderParametersBicepParamWriter.cs

```diff
@@ -79,26 +79,39 @@ private SyntaxBase GetValueForParameter(ParameterDeclarationSyntax syntax)
                 return defaultValue;
             }
 
+            if (syntax.Type is ObjectTypeSyntax objectTypeSyntax)
```

Comment:
See other comment on making this recursive - I don't think you need to do this.

Reply:
👍